src/evict/evict_inline.h

```diff
@@ -234,11 +233,11 @@ __wt_eviction_updates_needed(WT_SESSION_IMPL *session, double *pct_fullp)
 }
 
 /*
- * __wt_btree_dominating_cache --
+ * __wti_btree_dominating_cache --
```

Comment:
`__wti_btree_dominating_cache` is only used once in evict_lru.c and that's unlikely to change given the specificity of the function. If we move it to evict_lru.c that's one less function defined in exposed in extern.h.

Happy to push this out to a subsequent PR and keep the current PR scope small.

Reply:
Good catch, I will leave this to the next PR as well.